homeassistant/components/unifi_access/coordinator.py

```diff
@@ -0,0 +1,101 @@
+"""Coordinators for the UniFi Access integration."""
+
+from asyncio import CancelledError, Task
+from contextlib import suppress
+import logging
+
+import aiohttp
+import uiaccessclient
+
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryAuthFailed
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator, UpdateFailed
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class UniFiAccessDoorCoordinator(DataUpdateCoordinator[dict[str, uiaccessclient.Door]]):
+    """Handles refreshing door data."""
+
+    def __init__(
+        self, hass: HomeAssistant, api_client: uiaccessclient.ApiClient
+    ) -> None:
+        """Initialize the door coordinator class."""
+        super().__init__(
+            hass,
+            _LOGGER,
+            name="UniFi Access door",
+            always_update=False,
```

Comment:
why the always_update = False?

Reply:
The models returned by the client are comparable and as such as far as I can tell "always_update" is not needed, as we can limit updates to whenever the state actually changes. Is this not the correct way of doing that?